daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Typing.scala

```diff
@@ -765,18 +765,10 @@ private[validation] object Typing {
         TUpdate(TOptional(TContractId(TTyCon(retrieveByKey.templateId))))
     }
 
-    private def typeOfCommit(typ: Type, party: Expr, update: Expr): Type = {
+    private def checkCommit(typ: Type, party: Expr, update: Expr): Type = {
       checkType(typ, KStar)
       checkExpr(party, TParty)
       checkExpr(update, TUpdate(typ))
-      TScenario(typ)
-    }
-
-    private def typeOfMustFailAt(typ: Type, party: Expr, update: Expr): Type = {
```

Comment:
Was this just unused?

Reply:
I factorize the comment part of `typeOfCommit` and `typeOfMustFailAt`

Reply:
I revert this change as it is not related the the concern of the PR.